client/src/visualiser-src/common/AnimationProducer.ts

```diff
@@ -55,7 +55,7 @@ export default abstract class AnimationProducer {
     lines.forEach((line, i) => {
       const codeLine: CodeLine = {
         rectTarget: SVG()
-          .rect(2000, 18)
+          .rect(450, 18)
```

Comment:
The width of the code snippet window is based on the vertical screen width (30vw) and is not necessarily 450px for everyone, so the oblong/rectangle-with-rounded-corners won't look right in most cases. It might be better to just make it excessively wide. Maybe change it back to 2000?